fl4health/clients/numpy_fl_client.py

```diff
@@ -17,7 +20,16 @@
 
 
 class NumpyFlClient(NumPyClient):
-    def __init__(self, data_path: Path, device: torch.device) -> None:
+    def __init__(self, data_path: Path, device: torch.device, seed: Optional[int] = None) -> None:
+        if seed is None:
```

Comment:
I think we can encapsulate this into a function called something like `fix_random_seeds` that takes the optional seed and performs this logic and logging?